seed/utils/meters.py

```diff
@@ -37,11 +37,13 @@ class PropertyMeterReadingsExporter():
     settings are considered/used when returning actual reading magnitudes.
     """
 
-    def __init__(self, property_id, org_id, excluded_meter_ids):
+    def __init__(self, property_id, org_id, excluded_meter_ids, scenario_ids=None):
         self._cache_factors = None
         self._cache_org_country = None
 
-        self.meters = Property.objects.get(pk=property_id).meters.exclude(pk__in=excluded_meter_ids)
+        self.meters = Meter.objects.filter(
+            Q(property_id=property_id) | Q(scenario_id__in=scenario_ids)
+        ).exclude(pk__in=excluded_meter_ids)
```

Comment:
Had to add an extra condition for scenario-linked meters, since BuildingSync meters are no longer being linked to canonical properties